sdk/formrecognizer/test-resources.json

```diff
@@ -0,0 +1,192 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "baseName": {
+            "type": "String"
+        },
+        "endpointSuffix": {
+            "type": "string",
+            "defaultValue": "core.windows.net"
+        },
+        "testApplicationOid": {
+            "type": "string",
+            "metadata": {
+                "description": "The principal to assign the role to. This is application object id."
+            }
+        },
+        "formRecognizerBaseName": {
```

Comment:
`formRecognizerBaseName` is redundant, you can just use the `baseName` and and add any additional formatting in a variable.